schema/schema.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE catalog;
+
+CREATE TABLE items (
+  id BIGINT PRIMARY KEY,
+  genre_id BIGINT NOT NULL,
+  author_id BIGINT NOT NULL,
+  label_id BIGINT NOT NULL,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  FOREIGN KEY (genre_id) REFERENCES genres(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id),
+  FOREIGN KEY (label_id) REFERENCES labels(id)
+);
+
+CREATE TABLE music_albums (
+  id BIGINT PRIMARY KEY,
+  title VARCHAR(255) NOT NULL,
+  item_id BIGINT NOT NULL,
+  on_spotify BOOLEAN NOT NULL,
+  FOREIGN KEY (item_id) REFERENCES items(id)
+);
```

Comment:
- [ ] When creating the `Books`, `MusicAlbums`, and `Games` tables, kindly include the foreign keys that you listed in the `Items` table for each of these tables since they are inheriting from the table. 👍 😎